### DISCUSSION TITLE
Adaptability to the standard mopeka sensor?

### DISCUSSION THREAD

**Discussion by scorfman** [spbrogan/sensor.mopeka_pro_check#21] [General]
I notice in your notes that you worked with mopeka to get this straight. I have a couple standard ones and note that even the esphome integration says it will not work with the standard tank check, only the pro ones. Do you have any information you could share about that?

**Comment by spbrogan** [comment]
Nothing more than what i shared in both of those integrations.  Mopeka did not provide BLE advertisement format information for those sensors, and they claimed it would be difficult due to variations and different vendors they used.
